src/ompl/geometric/planners/rrt/RRTstar.h

```diff
@@ -148,7 +160,12 @@ namespace ompl
                 return delayCC_;
             }
 
-            /** \brief Controls whether the tree is pruned during the search. */
+            /** \brief Controls whether the tree is pruned during the search. This pruning removes
+                a vertex \e and \e its \e descendants if the lower-bounding estimate of a solution
+                constrained to pass the the \e vertex  is greater than the current solution.
+                As the ancestory of the descendent vertices can change at anytime, this means that the removed
+                descendents may actually be capable of later providing a better solution once
+                their incoming path passes through a different vertex (e.g., a change in homotopy class) */
```

Comment:
Expanded documentation on the limitations of the current pruning method.

I don't mean to say that there's anything wrong with this, I just wanted to make it clear where it can be overly aggressive. I guess I could make a safer pruning function (i.e., it only removes a vertex if all its descendents are to be removed too), but I don't know if that should replace this function or live in parallel.